### PR TITLE
Author image updates now officially supported.

### PR DESCRIPTION
**Pull request by jcairo**

### CODE REVIEW

File: author_api/test_authentication.py [hunk 1]
```diff
@@ -118,7 +117,6 @@ def test_registration(self):
 
     def test_registration_with_image(self):
         response = self.c.post('/author/registration', self.user_dict_with_img, format='multipart')
-        import pdb; pdb.set_trace()
```
Comment (xkb) [inline author_api/test_authentication.py:121]: I already removed this yesterday on master. Pull and rebase to avoid conflicts?

Comment (jcairo) [inline author_api/test_authentication.py:121]: You got it.

Comment (jcairo) [inline author_api/test_authentication.py:121]: Won't it auto merge?

Comment (xkb) [inline author_api/test_authentication.py:121]: Most likely. I just never trust Git's smart auto-merge.

Comment (jcairo) [inline author_api/test_authentication.py:121]: I think it'll do for a single line.

**Pull request merged**